opentracing-spring-web-autoconfigure/src/test/java/io/opentracing/contrib/spring/web/autoconfig/ServerTracingAutoConfigurationTest.java

```diff
@@ -62,26 +82,35 @@ public MockTracer tracer() {
     @Test
     public void testRequestIsTraced() {
         testRestTemplate.getForEntity("/hello", String.class);
-        Awaitility.await().until(reportedSpansSize(), IsEqual.equalTo(2));
-
-        assertThat(mockTracer.finishedSpans()).hasSize(2);
+        Awaitility.await().until(reportedSpansSize(), IsEqual.equalTo(1));
+        assertThat(mockTracer.finishedSpans()).hasSize(1);
```

Comment:
What is the reason for the change in span/trace number from 2 to 1?

Reply:
Previous there were 2 spans. There wasn't a controller so the first span is for missing controller and second span was for error controller.